packages/fiori/src/DynamicSideContent.hbs

```diff
@@ -0,0 +1,20 @@
+<div
+	class="{{classes.dynamicSideContentRoot}}"
```

Comment:
dynamicSideContent here is redundant as this is the component itself, so just `root` should be good

Reply:
Done